### PR TITLE
Expose nodata value as part of the core common metadata #136

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java [hunk 1]
```diff
@@ -73,6 +73,8 @@
  */
 package it.geosolutions.imageioimpl.plugins.tiff;
 
+import com.sun.media.imageioimpl.common.ImageUtil;
+import com.sun.media.imageioimpl.common.PackageUtil;
```
Comment (dromagnoli) [inline plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java]: Are these imports really needed?

File: plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java [hunk 2]
```diff
@@ -81,45 +83,26 @@
 import it.geosolutions.imageio.plugins.tiff.TIFFField;
 import it.geosolutions.imageio.plugins.tiff.TIFFImageReadParam;
 import it.geosolutions.imageio.stream.input.FileImageInputStreamExtImpl;
+import org.w3c.dom.Node;
```
Comment (dromagnoli) [inline plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java]: Is it used somewhere?

File: plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java (hunk 1 above)
Comment (aaime) [reply plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java]: Yes, they are needed down. IntelliJ "optimized" the imports in a different way than Eclipse, that's why you get them here... but they were already there.

Comment (dromagnoli) [reply plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java]: right sorry. Didn't notice the -import counterpart there below.

**Pull request merged**